library/src/main/java/br/com/goncalves/pugnotification/notification/Custom.java

```diff
@@ -63,21 +73,25 @@ private void setSmallIcon() {
     }
 
     public Custom background(@DrawableRes int resource) {
-        if (resource <= 0) {
-            throw new IllegalArgumentException("Resource ID Should Not Be Less Than Or Equal To Zero!");
+        if (resource == 0) {
```

Comment:
@kmalmur Isn't `-1` considered as a possible value and an invalid ResourceId?

Reply:
I would suggest to keep the '<='
https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/content/res/ResourceId.java#39

Reply:
Non negative check causes the lint issue, but the snippet you provided seems to do exactly what we want to do. Thanks